src/locale/locales/fr/messages.po

```diff
@@ -5250,31 +4878,31 @@ msgstr "Envoyer un fichier texte vers :"
 #: src/view/com/util/UserBanner.tsx:116
 #: src/view/com/util/UserBanner.tsx:119
 msgid "Upload from Camera"
-msgstr ""
+msgstr "Téléchargement à partir de l’appareil photo"
```

Comment:
It's a verb here. Also, `télécharger` is ambiguous (it means `download` most of the time, even if it's used sometimes for `upload`).

```suggestion
msgstr "Envoyer à partir de l’appareil photo"
```